clusters/app.ci/prow-monitoring/additional-scrape-configs_secret.yaml

```diff
@@ -42,6 +42,7 @@ stringData:
           - https://artifacts-rpms-openshift-origin-ci-rpms.apps.ci.l2s4.p1.openshiftapps.com/openshift-origin-v4.2/repodata/repomd.xml
           - https://vault.ci.openshift.org/v1/sys/health
           - https://hook-apps-ci.apps.ci.l2s4.p1.openshiftapps.com
+          - https://rhcos-redirector.ci.openshift.org/art/storage/releases/rhcos-4.7/47.83.202103251640-0/x86_64/meta.json
```

Comment:
This will enable alerts if
- the SSL certs of the site are expiring in 30 days.
- the site does not return 200 for a minute.